scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -465,3 +482,6 @@ def test_lnet(self):
     execute_command(self, cmd)
 
     lnet_ping(self)
+def test_libfabric(self):
+    search_libfabric_pkgs = ["libfabric"]
+    check_pkgs(self, search_lnet_pkgs)
```

Comment:
This is typo?
Should it be:
check_pkgs(self, search_libfabric_pkgs)
?

Reply:
incorporated